src/test/java/io/vertx/core/http/Http2ServerTest.java

```diff
@@ -2545,7 +2547,9 @@ public void onHeadersRead(ChannelHandlerContext ctx, int streamId, Http2Headers
         @Override
         public void onUnknownFrame(ChannelHandlerContext ctx, byte frameType, int streamId, Http2Flags flags, ByteBuf payload) {
           int s = status++;
```

Comment:
I detect that this code is problematic. According to the [Correctness (CORRECTNESS)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#correctness-correctness), [DLS: Overwritten increment (DLS_OVERWRITTEN_INCREMENT)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#dls-overwritten-increment-dls-overwritten-increment).
The code performs an increment operation (e.g., i++) and then immediately overwrites it. For example, i = i++ immediately overwrites the incremented value with the original value.